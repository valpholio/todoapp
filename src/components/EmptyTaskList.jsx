import React from 'react';

const EmptyTaskList = () => {
    return (
        <div>
            You todo list is empty. Please, add a new task, using the following button:
        </div>
    );
};

export default EmptyTaskList;