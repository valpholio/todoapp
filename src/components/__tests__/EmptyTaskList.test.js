import React from 'react';
import renderer from 'react-test-renderer';
import EmptyTaskList from '../EmptyTaskList.jsx';

describe('EmptyTaskList', () => {
    test('snapshot renders', () => {
        const component = renderer.create(<EmptyTaskList/>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
