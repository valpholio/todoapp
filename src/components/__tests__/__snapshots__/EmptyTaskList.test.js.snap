// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyTaskList snapshot renders 1`] = `
<div>
  The todo list is empty. Please, add a new task, using the following button:
</div>
`;
